### PR TITLE
refactor(console): update all logo uploaders

### PR DESCRIPTION
**Pull request by gao-sun**
## Summary

resolve LOG-9592

- allow upload standard connector logos
- update content according to the latest design
- update application branding section

image

image

## Testing

manually tested. a bug bash is scheduled.

## Checklist

- [ ] `.changeset`
- [ ] unit tests
- [ ] integration tests
- [x] necessary TSDoc comments

### CODE REVIEW

File: packages/console/src/components/ImageInputs/index.tsx [hunk 1]
```diff
@@ -40,6 +45,8 @@ export type ImageField<FormContext extends FieldValues> = {
 type Props<FormContext extends FieldValues> = {
   /** The condensed title when user assets service is available. */
   readonly uploadTitle: React.ComponentProps<typeof FormField>['title'];
+  /** The tooltip to show for all the fields. */
```
Comment (xiaoyijun) [inline packages/console/src/components/ImageInputs/index.tsx:48]: nit:
`!isUserAssetServiceReady`: for all input fileds
`isUserAssetServiceReady`: for the form field section

**Review by xiaoyijun — APPROVED** [review]
Overall LGTM.

File: packages/console/src/pages/ApplicationDetails/ApplicationDetailsContent/Branding/index.tsx [hunk 2]
```diff
@@ -85,8 +92,72 @@ function Branding({ application, isActive }: Props) {
...
+
+  // When enabling branding for the first time, fill the default color values to ensure the form
+  // is valid; otherwise, directly save the form will be a no-op.
+  useEffect(() => {
+    if (isBrandingEnabled && Object.values(color).filter(Boolean).length === 0) {
+      setValue('color', { primaryColor: '#000000', darkPrimaryColor: '#000000' });
+    }
+  }, [color, isBrandingEnabled, setValue]);
+
+  const NonThirdPartyBrandingForm = useCallback(
```
Comment (charIeszhao) [inline packages/console/src/pages/ApplicationDetails/ApplicationDetailsContent/Branding/index.tsx:112]: Should be a `useMemo` instead of `useCallback`

**Review by darcyYe — APPROVED** [review]

File: packages/console/src/components/ImageInputs/index.tsx (hunk 1 above)
Comment (gao-sun) [reply packages/console/src/components/ImageInputs/index.tsx:48]: they are included in "all the fields", right?

Comment (gao-sun) [reply packages/console/src/components/ImageInputs/index.tsx:48]: LOG-9596

**Pull request merged**